vars/all.yml

```diff
@@ -1,5 +0,0 @@
----
-rax_name: ansible_pyvideo
-rax_flavor: 3 # 1 gig standard instance
-#598a4282-f14b-4e50-af4c-b3e52749d9f9 Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)
-rax_image: 598a4282-f14b-4e50-af4c-b3e52749d9f9
```

Comment:
I removed this from the repo. You'll want to copy it down and stick it in your `all.yml` file.